cpmpy/solvers/gurobi.py

```diff
@@ -83,9 +83,9 @@ def __init__(self, cpm_model=None, subsolver=None):
         Arguments:
         - cpm_model: a CPMpy Model()
         """
-        if not self.supported():
-            raise Exception(
-                "CPM_gurobi: Install the python package 'gurobipy' and make sure your licence is activated!")
+        # if not self.supported():
```

Comment:
We want to throw this exception right, is this still leftover from testing?

Reply:
Yes, this is leftover from testing... Licence was expired ;)